atmo/coordinator/executor/executor.go

```diff
@@ -256,3 +261,29 @@ func (e *Executor) Metrics() (*rt.ScalerMetrics, error) {
 
 	return &metrics, nil
 }
+
+func connectStaticPeers(log *vlog.Logger, g *grav.Grav, opts *options.Options) error {
+	count := 0
+	var err error
+
+	epts := strings.Split(opts.StaticPeers, ",")
+
+	for _, e := range epts {
+		log.Debug("connecting to static peer", e)
+
+		for count < 10 {
+			if err = g.ConnectEndpoint(e); err != nil {
+				log.Error(errors.Wrap(err, "failed to ConnectEndpoint, will retry"))
+				count++
+
+				time.Sleep(time.Second * 3)
+			} else {
+				break
+			}
+		}
+	}
+
+	log.Error(errors.Wrap(err, "failed to ConnectEndpoint, retries exceeded"))
```

Comment:
This looks like we'd get a log message no matter what, is that true?

Reply:
Fixed!